doc/ap_verify_hits2015/index.rst

```diff
@@ -0,0 +1,40 @@
+.. _ap_verify_hits2015-package:
+
+##################
+ap_verify_hits2015
+##################
+
+The ``ap_verify_hits2015`` package represents a small subset of the DECam `HiTS <http://astroinf.cmm.uchile.cl/the-high-cadence-transient-survey/>`_ survey.
+It is a good example dataset for difference imaging with DECam.
+
+Project info
+============
+
+Repository
+   https://github.com/lsst/ap_verify_hits2015
+
+.. Datasets do not have their own (or a collective) Jira components; by convention we include them in ap_verify
+
+Jira component
+   `ap_verify <https://jira.lsstcorp.org/issues/?jql=project %3D DM %20AND%20 component %3D ap_verify %20AND%20 text ~ "hits2015">`_
+
+Dataset Contents
+================
+
+This package provides complete data for three fields from the 2015 HiTS survey.
```

Comment:
Add a link to the HiTS paper: http://iopscience.iop.org/article/10.3847/0004-637X/832/2/155/meta

Reply:
Is that really better than a website link?

Reply:
Both/and can't hurt here. The paper describes the survey better than any website I've found, and also explains what region of the sky each field corresponds to, etc.